### ISSUE TITLE
Add a delay to Nest Camera motion clear event

### ISSUE THREAD

**Issue by rs** [KraigM/homebridge-nest#103]
I would like to use the Nest Camera motion event to control lights, but the clear event is triggered as soon as no motion is detected anymore, which is way to fast for my use-case.

It would be awesome if we define a delay before the motion state goes back to `clear` when motion is no longer detected.

**Comment by EnduroFever** [comment]
Agreed!  Even a single value for all devices to follow would be nice.  I use this capability with my Elgato motion sensors and would use it the same way with Nest if it were possible.
